MaxText/layers/attentions.py

```diff
@@ -49,55 +49,15 @@
 LENGTH = common_types.LENGTH
 HEAD = common_types.HEAD
 D_KV = common_types.D_KV
+DEFAULT_MASK_VALUE = -0.7 * float(jnp.finfo(jnp.dtype("float32")).max)
```

Comment:
Probably consider rename to Pallas_Default_.... etc as this is pallas tpu kernel specific.

Reply:
But we're using this in all dot_product attention, right?

Reply:
Ok.